python/tools/dht/tests.py

```diff
@@ -17,6 +17,15 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import networkx as nx
+import subprocess
+import re
+
+from matplotlib.ticker import FuncFormatter
+import math
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FuncFormatter
```

Comment:
Update done :) Thanks for the review.
And yup in fact that was a trouble will merging ( i suppose )